spec/altair/syncaggregate.go

```diff
@@ -70,12 +70,6 @@ func (s *SyncAggregate) unpack(syncAggregateJSON *syncAggregateJSON) error {
 	if err != nil {
 		return errors.Wrap(err, "invalid value for sync committee bits")
 	}
-	if len(syncCommitteeBits) < syncCommitteeSize/8 {
-		return errors.New("sync committee bits too short")
-	}
-	if len(syncCommitteeBits) > syncCommitteeSize/8 {
-		return errors.New("sync committee bits too long")
-	}
```

Comment:
Not a fan of losing these checks (and those in syncommittee.go).  You could move syncCommitteeSize into the service struct, and obtain it from a call to `Spec()` on startup (I think it's fair to assume that this value won't change in the lifetime of the connection, so it doesn't have to be refreshed).

Reply:
I don't think this is possible here :(
It'd be great to have a reference to the Service in this function as it'd allow more spec conformance checks.
But it's following the standard JSON marshaler interface (UnmarshalJSON) and cannot be extended.

Reply:
I think that there is a way this could be done, but it would be pretty ugly so for now I'm okay with losing these checks.